libraries/Microsoft.Bot.Builder.Core.Extensions/MemoryStorage.cs

```diff
@@ -37,9 +37,9 @@ public Task Delete(string[] keys)
             return Task.CompletedTask;
         }
 
-        public Task<IEnumerable<KeyValuePair<string, object>>> Read(string[] keys)
+        public Task<IDictionary<string, object>> Read(string[] keys)
         {
-            var storeItems = new List<KeyValuePair<string, object>>(keys.Length);
+            IDictionary<string, object> storeItems = new Dictionary<string, object>(keys.Length);
```

Comment:
The codebase seems to be `var` based, so I think we would want to stay consistent with that.

Reply:
Fixed. I don't know why I was so fixated on the ".Add(KeyValuePair<>)" overload.